### PR TITLE
feat: Throw an Error if package.json has duplicate "repository" key

### PR DESCRIPTION
**Pull request by gr2m**
addresses bad DX reported in #1655 /cc @kentcdodds @travi

### CODE REVIEW

**Review by travi — APPROVED** [review]
one typo, but otherwise looks great 👍🏼

**Comment by gr2m** [comment]
where's the typo?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ Before pushing your code changes make sure there are no linting errors with `npm
 
 ### Tests
 
-Running the integration test requires you to install [Docker](https://docs.docker.com/engine/installation) on your machine.
+Running the integration test requires you to install [Docker](https://docs.docker.com/engine/installation) on your machine. Note: the test assume that running `git init` will create a `master` branch by default. If your local `git` is configured differently (see `init.defaultBranch`), change it temporarily when running the tests.
```
Comment (gr2m) [inline CONTRIBUTING.md:232]:
```suggestion
Running the integration test requires you to install [Docker](https://docs.docker.com/engine/installation) on your machine. Note: the tests assume that running `git init` will create a `master` branch by default. If your local `git` is configured differently (see [`init.defaultBranch`](https://github.blog/2020-07-27-highlights-from-git-2-28/#introducing-init-defaultbranch)), change it temporarily when running the tests.
```

**Comment by travi** [comment]
> where's the typo?

looks like you found my comment. surprised that it didnt show up in this thread. must be because i still approved 🤷🏼

**Review by kentcdodds — APPROVED** [review]
Nice!

**Comment by travi** [comment]
i reverted this change for now in order to give us time to understand #1657

**Pull request merged**